src/overlays/actors/ovl_En_Ko/z_en_ko.c

```diff
@@ -47,93 +70,1248 @@ static ColliderCylinderInit D_80A9A100 = {
+                        this->actor.textId = (globalCtx->msgCtx.choiceIndex != 0)
+                                                 ? (globalCtx->msgCtx.choiceIndex == 1) ? 0x103A : 0x103B
+                                                 : 0x1039;
+                        func_8010B720(globalCtx, this->actor.textId);
+                        break;
+                    case 0x103E:
+                        this->actor.textId = (globalCtx->msgCtx.choiceIndex == 0) ? 0x103F : 0x1040;
+                        func_8010B720(globalCtx, this->actor.textId);
+                        break;
+                    case 0x10B7:
+                        gSaveContext.infTable[11] |= 0x1000;
+
+                    case 0x10B8:
+                        this->actor.textId = (globalCtx->msgCtx.choiceIndex == 0) ? 0x10BA : 0x10B9;
+                        return (globalCtx->msgCtx.choiceIndex == 0) ? 2 : 1;
+                }
+                return 1;
+            }
+            break;
+        case 6:
+            if (func_80106BC8(globalCtx) != 0) {
+                return 3;
+            }
+    }
+    return 1;
+}
+
+s32 EnKo_GetForestQuestState(EnKo* this) {
+    s32 result;
+    if (LINK_IS_CHILD) {
+        // Obtained Zelda's Letter
+        if (gSaveContext.eventChkInf[4] & 1) {
+            return ENKO_FQS_CHILD_SARIA;
+        }
+        if (CHECK_QUEST_ITEM(QUEST_KOKIRI_EMERALD)) {
+            return ENKO_FQS_CHILD_STONE;
+        }
+        return ENKO_FQS_CHILD_START;
+    }
+
+    if (CHECK_QUEST_ITEM(QUEST_MEDALLION_FOREST)) {
+        result = ENKO_FQS_ADULT_SAVED;
+    } else {
+        result = ENKO_FQS_ADULT_ENEMY;
+    }
+    return result;
+}
+
+f32 func_80A97BC0(EnKo* this) {
+    f32 D_80A9A62C[13][5] = { /* ENKO_TYPE_CHILD_0    */ { 0.0f, 0.0f, 0.0f, -30.0f, -20.0f },
+                              /* ENKO_TYPE_CHILD_1    */ { 0.0f, 0.0f, 0.0f, -20.0f, -10.0f },
+                              /* ENKO_TYPE_CHILD_2    */ { 0.0f, 0.0f, 0.0f, -30.0f, -20.0f },
+                              /* ENKO_TYPE_CHILD_3    */ { -10.0f, 10.0f, 10.0f, -10.0f, -30.0f },
+                              /* ENKO_TYPE_CHILD_4    */ { 0.0f, 0.0f, 0.0f, -10.0f, -20.0f },
+                              /* ENKO_TYPE_CHILD_5    */ { 0.0f, 0.0f, 0.0f, -20.0f, -20.0f },
+                              /* ENKO_TYPE_CHILD_6    */ { 0.0f, 0.0f, 0.0f, -10.0f, -20.0f },
+                              /* ENKO_TYPE_CHILD_7    */ { 10.0f, 10.0f, 10.0f, -60.0f, -20.0f },
+                              /* ENKO_TYPE_CHILD_8    */ { -10.0f, -10.0f, -20.0f, -30.0f, -30.0f },
+                              /* ENKO_TYPE_CHILD_9    */ { -10.0f, -10.0f, -10.0f, -40.0f, -40.0f },
+                              /* ENKO_TYPE_CHILD_10   */ { 0.0f, 0.0f, 0.0f, -10.0f, -20.0f },
+                              /* ENKO_TYPE_CHILD_11   */ { -10.0f, -10.0f, -20.0f, -30.0f, -30.0f },
+                              /* ENKO_TYPE_CHILD_FADO */ { 0.0f, 0.0f, 0.0f, -20.0f, -20.0f } };
```

Comment:
does a trailing comma help the formatting not indent everything to the right?
same with `D_80A9A730`